### PR TITLE
GarbageCollector section is not clear

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1704598

### CODE REVIEW

**Comment by sunilcio** [comment]
LGTM

**Comment by sunilcio** [comment]
@mburke5678 I just checked your question in bz. Garbage collection for containers is enabled by default. For image garbage collection, user need to manually enable it. So we need to modify the statement to reflect that in image garbage collection.

File: admin_guide/garbage_collection.adoc [hunk 1]
```diff
@@ -15,9 +15,9 @@ toc::[]
 The {product-title} node performs two types of garbage collection:
 
 * xref:container-garbage-collection[Container garbage collection] Removes
```
Comment (kalexand-rh) [inline admin_guide/garbage_collection.adoc:17]: I'd add a : after the xref.

**Review by kalexand-rh — COMMENTED** [review]
I'd consider moving each of the notes before their respective code blocks. It's not in the section that you changed, but I'm a little confused how we have default parameters for the image garbage collection if it's not enabled by default.

**Comment by mburke5678** [comment]
@kalexand-rh Thanks for the tips. I moved the notes and they do look better. 

@sunilcio I removed the references to the default values for the image garbage collection parameters based on Kathryn's comment above. It seems non-intuitive to say a parameter is not default but has default values. (It does seems that the parameters do have defaults.). Let me know if you think I should put them back.

**Comment by sunilcio** [comment]
I agree, looks better now. LGTM

**Pull request merged**